Mathlib/Tactic/Finiteness.lean

```diff
@@ -25,7 +25,7 @@ open Lean Elab Tactic in
 /-- A version of the positivity tactic for use by `aesop`. -/
 @[aesop safe tactic (rule_sets := [finiteness])]
```

Comment:
You should use `add_aesop_rules` here if possible, e.g.

```
add_aesop_rules safe tactic (rule_sets := [finiteness]) (by positivity)
```

Besides being shorter, this ensures that `aesop?` works.